docs/src/main/tut/faq.md

```diff
@@ -16,7 +16,7 @@ section: "faq"
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [How can I help?](#contributing)
 
-## What imports do I need?<a id="what-imports" href="#what-imports"></a>
+## <a id="what-imports" href="#what-imports"></a>What imports do I need?
```

Comment:
@mikejcurry ah I see. That makes sense. Let's go with your approach, and hopefully someone will be annoyed enough by the jumping text that they'll fix it for us :P